### PR TITLE
BAN-155: POC Stencil Puppeteer visual regression tests

### PR DESCRIPTION
**Pull request by chavlji**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/BAN-155)

**Don't merge!** It's just a POC.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,14 @@ yarn create:component zen-{component-name}
 yarn test
 ```
 
+#### E2E Testing (Puppeteer)
+TODO: **test:e2e can only be run using npm!**
```
Comment (mplesec) [inline README.md:39]: https://github.com/ionic-team/stencil/issues/1609#issuecomment-755362126

File: screenshot/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+images
+builds
+compare.html
```
Comment (mplesec) [inline screenshot/.gitignore:3]: Hm, can we ignore the `screenshot` folder instead?

**Review by mplesec — APPROVED** [review]
Awesome

File: README.md [hunk 3]
```diff
@@ -35,6 +35,18 @@ yarn create:component zen-{component-name}
...
 ```
 
+#### E2E Testing (Puppeteer)
+**Note:** Be sure to use `yarn run` when running this command. Otherwise you will get `No tests found` error.
+```
+yarn run test:e2e
+```
+
+#### Visual regression testing (Puppeteer)
+**Note:** Be sure to use `yarn run` when running this command. Otherwise you will get `No tests found` error.
```
Comment (mplesec) [inline README.md:45]: Don't think these are necessary, it's clear how the command is supposed to be run.

File: .gitignore [hunk 4]
```diff
@@ -30,3 +30,4 @@ $RECYCLE.BIN/
 Thumbs.db
 UserInterfaceState.xcuserstate
 .env
+/screenshot
```
Comment (mplesec) [inline .gitignore:33]: NIT: why the `/`?

Comment (chavlji) [reply .gitignore:33]: To explicitly ignore only root screenshot folder

**Review by f-jauregui — APPROVED** [review]

**Comment by chavlji** [comment]
We've decided to continue with Cypress, so this PR is irrelevant now.

**Pull request closed without merging**